README.md

```diff
@@ -20,4 +20,4 @@ Add your answers to this markdown file.
 
 2. The time compelexity could only be expected to be log(10000) if the tree is balanced, but there is no way to know the exact time it would take to find the same element. It would depend on many different factors such as hardware and the data that was put into the tree. 
 
-3. One reason this could have happened is because the tree is unbalanced, which causes the computer to do more work in search of the element. Another reason may be that the size of the array is too big for one memory allocation, so the computer has to use a larger but slower memory unit to store the tree. A third reason this could happen is because the search function for the element in the tree is written enefficiently and calls unnecessary function calls or iterations. 
+3. One reason this could have happened is because the tree is unbalanced to the point where it is basically a linked list while also iterating through the list twice, which causes the computer to do more work in search of the element. Another reason may be that . A third reason this could happen is because the search function for the element in the tree is written enefficiently and calls unnecessary function calls or iterations.
```

Comment:
Unbalanced tree can't account for a difference that large, as discussed in class.

Reply:
If the search iterated through the list twice, wouldn't that make the runtime n^2?

Reply:
Why would it go through the list twice?

Reply:
There must be unnecessary calls or iterations to make the function take so long.

Reply:
Ok, so the implementation isn't as efficient as it could be. That doesn't seem to have anything to do with whether the tree is balanced or not though.